satpy/tests/reader_tests/test_viirs_l2.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
These lines are redundant and don't add anything.

Reply:
Removed in 526b4549f